travvy-server/src/models/Users.js

```diff
@@ -0,0 +1,22 @@
+// Sequelize model for table that is called "users"
+module.exports = (sequelize, DataTypes) => {
+    const User = sequelize.define('User', {
+      Email: {
+        type: DataTypes.TEXT,
+        field: 'email',
+        primaryKey: true
+      },
+      Name: DataTypes.TEXT,
+      Password: DataTypes.TEXT,
+      Age: DataTypes.INTEGER,
+      Gender: DataTypes.TEXT,
+      Instagram_Username: DataTypes.TEXT
+    }, {
+      tableName: 'Users',
+      timestamps: false
+    }
+    )
+  
+    return User
```

Comment:
This looks great Lucas - I look forward to using this file to link the sign up page and the users, so the users input to the database!